cmake/CMakeLists.txt

```diff
@@ -340,11 +341,12 @@ if (WIN32)
 
     # set linker flags to minimize the binary size.
     if (MSVC)
-      foreach(type EXE SHARED)    
+      foreach(type EXE SHARED)
         set(CMAKE_${type}_LINKER_FLAGS_RELWITHDEBINFO "${CMAKE_${type}_LINKER_FLAGS_RELWITHDEBINFO} /OPT:REF")
```

Comment:
Use /OPT:REF,ICF,LBR for more optimization. This is usually the default, but not in our case. Read here for details: https://docs.microsoft.com/en-us/cpp/build/reference/opt-optimizations?view=vs-2017.

Reply:
Done.